From 6fefe7fe50225c23150abbb6acbc78314397705f Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 19 Jan 2024 06:07:43 +0100
Subject: [PATCH] db: Update db8.groups.json for test API's

Fixes:

WARNING: Found 2 group(s) that appear only in api-permissions.d, consider define them in groups.d
WARNING: === LIST BEGIN: Groups used in api-permissions.d but not defined in groups.d ===
WARNING:   database.test

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/dynamic/db8.groups.json | 5 +++--
 files/sysbus/static/db8.groups.json  | 5 +++--
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/files/sysbus/dynamic/db8.groups.json b/files/sysbus/dynamic/db8.groups.json
index 7e4bed8..8ff9d8e 100644
--- a/files/sysbus/dynamic/db8.groups.json
+++ b/files/sysbus/dynamic/db8.groups.json
@@ -1,5 +1,5 @@
 {
-  "allowedNames": ["com.palm.db","com.palm.tempdb","com.webos.mediadb","com.webos.epgdb", "com.webos.keymgrdb"],
+  "allowedNames": ["com.palm.db","com.palm.tempdb","com.webos.mediadb","com.webos.epgdb", "com.webos.keymgrdb", "com.webos.db8.test.*"],
   "database.operation":["dev"],
   "database.profiling":["dev"],
   "database.management":["oem"],
@@ -14,5 +14,6 @@
   "mediadatabase.management":["oem"],
   "keymgrdatabase.operation":["oem"],
   "keymgrdatabase.profiling":["oem"],
-  "keymgrdatabase.management":["oem"]
+  "keymgrdatabase.management":["oem"],
+  "database.test":["dev"]
   }
\ No newline at end of file
diff --git a/files/sysbus/static/db8.groups.json b/files/sysbus/static/db8.groups.json
index d73d812..01d6ec0 100644
--- a/files/sysbus/static/db8.groups.json
+++ b/files/sysbus/static/db8.groups.json
@@ -1,5 +1,5 @@
 {
-  "allowedNames": ["com.palm.db","com.palm.tempdb","com.webos.mediadb","com.webos.epgdb", "com.webos.keymgrdb"],
+  "allowedNames": ["com.palm.db","com.palm.tempdb","com.webos.mediadb","com.webos.epgdb", "com.webos.keymgrdb", "com.webos.db8.test.*"],
   "database.operation":["dev"],
   "database.profiling":["dev"],
   "database.management":["oem"],
@@ -14,5 +14,6 @@
   "mediadatabase.management":["oem"],
   "keymgrdatabase.operation":["oem"],
   "keymgrdatabase.profiling":["oem"],
-  "keymgrdatabase.management":["oem"]
+  "keymgrdatabase.management":["oem"],
+  "database.test":["dev"]
 }
\ No newline at end of file
